src/bin/travis/testCentaurTes.sh

```diff
@@ -42,7 +42,7 @@ mysql -u root -e "GRANT ALL PRIVILEGES ON cromwell_test . * TO 'travis'@'localho
 WORKDIR=$(pwd)
 
 sbt assembly
-CROMWELL_JAR=$(find "$(pwd)/target/scala-2.11" -name "cromwell-*.jar")
+CROMWELL_JAR=$(find "$(pwd)/target/scala-2.12" -name "cromwell-*.jar")
```

Comment:
which reminds me we should check what the release WDL is expecting WRT 2.11 / 2.12